backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java

```diff
@@ -651,4 +630,15 @@ private void throwErrorIfEmpty(Stream<?> stream, String errorMessage) {
       throw new IllegalGraphqlArgumentException(errorMessage);
     }
   }
+
+  private User getUserOrThrowError(String username, String errorMessage) {
+    var searchUsersStream =
+        _client.listUsers(null, null, generateLoginSearchTerm(username), null, null).stream();
+    var user = searchUsersStream.findFirst();
+    if (user.isEmpty()) {
+      var qUsersStream = _client.listUsers(username, null, null, null, null).stream();
```

Comment:
is there a convention / meaning for prefixing the variable here with `q`? Asking for curiosity!

Reply:
That's just the name of the parameter in [okta's documentation](https://developer.okta.com/docs/reference/api/users/#content-type-header-fields-2:~:text=FALSE-,q,FALSE,-limit).